exercises/practice/transpose/.meta/config.json

```diff
@@ -1,10 +1,32 @@
 {
   "blurb": "Take input text and output it transposed.",
-  "authors": [],
+  "authors": [
+    {
+      "github_username": "andrej-makarov-skrt"
+    }
+  ],
+  "contributors": [
+    {
+      "github_username": "SaschaMann"
+    },
+    {
+      "github_username": "kytrinyx"
```

Comment:
> Bulk update PRs made by ErikSchierboom or kytrinx to update the track

I believe this bulk PR may be counted by accident: https://github.com/exercism/julia/pull/75

Katrina is listed as contributor here but her only change to `transpose` was the bulk PR above, see https://github.com/exercism/julia/commits/v2-final-version/exercises/transpose